app/views/admins/_form.html.erb

```diff
@@ -1,109 +1,89 @@
 
-        <%= access_level_select(form, current_admin.permitted_access_levels,
-            {
-                required: true,
-                disabled: !current_admin.modify_access_level?,
-                current_access_level: admin.access_level
-            }) %>
+      <%= access_level_select(form, current_admin.permitted_access_levels,
+          {
+              required: true,
+              disabled: !current_admin.modify_access_level?,
+              current_access_level: admin.access_level
+          }) %>
 
-        <%= form.label "Facility access *" %>
-        <div id="facility-access-power-user" class=<%= "hidden" unless admin.power_user? %>>
-          Admin has access to all facilities in the system
-        </div>
+      <%= form.label "Facility access *" %>
+      <div id="facility-access-power-user" class=<%= "hidden" unless admin.power_user? %>>
+        Admin has access to all facilities in the system
+      </div>
 
-        <div id="facility-access" class=<%= "hidden" if admin.power_user? %>>
-          <div class="card access-tree">
-            <div id="select-all-facilities" hidden="hidden">
-              <%= form.check_box :select_all_facilities, id: "select-all-facilities-input" %>
-              <div class="counter" id="total-selected-facilities"></div>
-            </div>
+      <div id="facility-access" class=<%= "hidden" if admin.power_user? %>>
+        <div class="card access-tree">
+          <div id="select-all-facilities" hidden="hidden">
+            <%= form.check_box :select_all_facilities, id: "select-all-facilities-input" %>
+            <div class="counter" id="total-selected-facilities"></div>
+          </div>
 
-            <%= render_async(access_tree_admin_path(admin, :edit), error_message: access_tree_load_error) do %>
-              <div class="spinner-wrapper">
-                <div class="spinner spinner-border text-primary" role="status">
-                  <span class="sr-only">Loading...</span>
-                </div>
+          <%= render_async(access_tree_admin_path(admin, :edit), error_message: access_tree_load_error) do %>
+            <div class="spinner-wrapper">
+              <div class="spinner spinner-border text-primary" role="status">
+                <span class="sr-only">Loading...</span>
               </div>
-            <% end %>
-          </div>
+            </div>
+          <% end %>
         </div>
+      </div>
 
-        <%= form.submit "Send Invite", class: "btn btn-primary", id: "form-submit", hidden: :hidden %>
-      <% end %>
-    </div>
+      <%= form.submit "Send Invite", class: "btn btn-primary", id: "form-submit", hidden: :hidden %>
+    <% end %>
   </div>
+</div>
 
-  <script>
-    new AdminAccessEdit("facility-access").initialize()
-  </script>
-<% else %>
-  <%= javascript_include_tag "standalone/react_components" %>
-  <% user_permissions = current_admin.user_permissions.pluck(:permission_slug) %>
-  <%= react_component "InviteAdminForm",
```

Comment:
Ooh, can we delete some react files now too?

Reply:
👻